apps/portal/js/store-manager.js

```diff
@@ -62,34 +61,80 @@ var getAsset, getAssets, addAsset, deleteAsset, getDashboardsFromRegistry;
         var userRoles = um.getRoleListOfUser(ctx.username);
 
         var dashboards = getDashboardsFromRegistry(start, count);
-        if (!dashboards) {
+        var superTenantDashboards = null;
+        var superTenantRegistry = null;
+
+        if (ctx.domain != superDomain) {
+            superTenantRegistry = new carbon.registry.Registry(server, {
+                system: true,
+                tenantId: carbon.server.superTenant.tenantId
+            });
+            superTenantDashboards = superTenantRegistry.content(registryPath(), {
+                start: start,
+                count: count
+            });
+        }
+
+        if (!dashboards && !superTenantDashboards) {
             return [];
         }
-        var allDashboards = [];
-        dashboards.forEach(function (dashboard) {
-            allDashboards.push(JSON.parse(registry.content(dashboard)));
-        });
 
         var userDashboards = [];
-        allDashboards.forEach(function (dashboard) {
-            var permissions = dashboard.permissions,
-                data = {
-                    id: dashboard.id,
-                    title: dashboard.title,
-                    description: dashboard.description,
-                    pagesAvailable: dashboard.pages.length > 0,
-                    editable: true
-                };
-
-            if (utils.allowed(userRoles, permissions.editors)) {
-                userDashboards.push(data);
-                return;
-            }
-            if (utils.allowed(userRoles, permissions.viewers)) {
-                data.editable = false;
-                userDashboards.push(data);
+        if (dashboards) {
+            var allDashboards = [];
+            dashboards.forEach(function (dashboard) {
+                allDashboards.push(JSON.parse(registry.content(dashboard)));
+            });
+
+            allDashboards.forEach(function (dashboard) {
```

Comment:
This is again being repeated in line# 118, can we move this block after we have iterated the super tenant space as well and get all list of dashboards?